common/lib/common-definitions/src/events.ts

```diff
@@ -5,6 +5,8 @@
 
 /**
  * Base interface for event emitters.
+ *
+ * @deprecated - Use the core-interfaces package instead.
```

Comment:
Nit: we could provide a somewhat better end user experience here by linking to the replacement in the new package. E.g.
```suggestion
 * @deprecated - Use the {@link @fluidframework/core-interfaces#IEvent} instead.
```

Reply:
Also note: `@deprecated` should not be followed by a `-`. As written, intellisense / generated docs will read as "- Use ..."

Reply:
Ugh, this inconsistency in the tsdoc spec gets me every time. Do we have a lint rule?

Reply:
> Nit: we could provide a somewhat better end user experience here by linking to the replacement in the new package. E.g.
> 
> ```suggestion
> 
>  * @deprecated - Use {@link @fluidframework/core-interfaces#IEvent} instead.
> 
> ```

I got lazy. I never remember the syntax, so every time I add a link I have to look up the syntax. I'll clean these up.